docs/whats-new/zowe-v3-frequently-asked-questions.md

```diff
@@ -0,0 +1,125 @@
+
+2. When will Zowe V3 be available for preview?
+  
+   Currently, Zowe V3 pre-release versions are available for the PAX, Zowe CLI, Zowe Client Node.js SDKs, Zowe Explorer for Visual Studio Code, and the Zowe IntelliJ Plug-in.
+
+## Extender questions
+
+### API Mediation Layer
+
+1. Do we need to move away from passtickets as the method of authentication to the API Mediation Layer?
+    
+   No, we will continue to support passtickets. However, passtickets will be deprecated in Zowe V3 and are no longer recommend because configuration can be inefficient. For every service that uses them, a user must configure a passticket, for example.
+
+2. With the introduction of the new cloud gateway, how is the configuration going to change?
+
+   The cloud gateway configuration for V3 will move to the currently used gateway configuration space (for example, zowe.components.gateway).The configuration that was Zuul specific will be removed and replaced with the Spring Cloud Gateway configuration.
+
+3. How does client certificate authentication work in Zowe V3?
+
+   Northbound authentication accepts client certificates. The API Mediation Layer then transforms the client certificate to another method of authentication (such as a JWT token, SAF IDT, or passticket) and this new method is then accepted by southbound services. Currently, we do not plan to support authentication with client certificates to z/OSMF.
+
+4. Will you identify deprecated functionality such as passtickets?
+
+    Passtickets will be deprecated in V3, but they will still be supported, possibly even through to Zowe V4.
+
+    We plan to identify all deprecated functionality. We will announce how this functionality will be identified in the near future.
+
+5. How do you intend to work with the the bypass scheme?
+
+   The bypass scheme remains and you can still claim conformance for services that only provide public endpoints that do not require authentication.
+
+6. Do the dependency changes mean that I need Java 17 to run Zowe V3?
+
+   Yes. Java 17 is required to run the API Mediation Layer in Zowe V3.
+
+### Zowe Explorer for Visual Studio Code
+
+1. Will the functionality to convert Zowe V1 profiles to team configuration remain available for Zowe V3?
+
+Yes. However, users will not be able to use Zowe V1 profiles to connect to services on the mainframe.
+
+2. Will the APIs that Zowe CLI provides for extenders remain the same?
+
+   In broad terms, yes. However, some of the functionality that was available in Zowe V2 has been changed (or removed). The changes are included in these [CLI](https://ibm.ent.box.com/s/vqu92d82b4wk0i6fupo8glbrxvufn4zw) and [Imperative](https://github.com/zowe/imperative/issues/970) lists of breaking changes.
+
+### Zowe Application Framework, ZSS
+
+1. Is it possible to run 32-bit plug-ins at the same time as 64-bit plug-ins with ZSS?
+
+   No. ZSS runs in either 32-bit or 64-bit mode, which means it can accommodate only one type of plug-in.
```

Comment:
Strange but true: 31-bit. z/OS is has 31-bit or 64-bit programs. There is no 32-bit.